guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,12 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing post-upgrade tasks
 
+* Optional: If you are using the {oVirt} compute resource, you can pre-migrate your data by executing the `foreman-rake ovirt:drop` task on your {ProjectServer}.
+You can set `OVIRT_PRETEND=true` to preview the changes.
+The {oVirt} compute resource will be officially phased out in a future release and this migration will be triggered automatically during a future upgrade.
+ifndef::satellite[]
+{Team} recommends also removing the {oVirt} package by executing `{project-package-remove} foreman-ovirt` to prevent users from adding a new {oVirt} compute resource.
```

Comment:
```suggestion
{Team} also recommends removing the {oVirt} package by executing `{project-package-remove} foreman-ovirt` to prevent users from adding a new {oVirt} compute resource.
```
"Also" should come before a single action verb. :)